docs/cli/commands/deploy.mdx

```diff
@@ -30,8 +30,17 @@ When successfully built, the deployment will be promoted to production and will
 
 You can learn more about [preview and production deployments](/deployments) in the deployments documentation.
 
+## Build logs
```

Comment:
```suggestion
### Build logs
```

Based on the preview docs url - since on there it looks like the next heading (Flags) is a subheader of this one